src/System.Collections/tests/Performance/Perf.Dictionary.cs

```diff
@@ -111,9 +94,9 @@ public void GetItem(int size)
                 using (iteration.StartMeasurement())
                     for (int i = 0; i <= 10000; i++)
                     {
-                        retrieved = dict["key1"]; retrieved = dict["key2"]; retrieved = dict["key3"];
-                        retrieved = dict["key4"]; retrieved = dict["key5"]; retrieved = dict["key6"];
-                        retrieved = dict["key7"]; retrieved = dict["key8"]; retrieved = dict["key9"];
+                        retrieved = dict[1]; retrieved = dict[2]; retrieved = dict[3];
```

Comment:
Seems like it would depend on what we're trying to test... we might want both a test when there aren't many collisions and a test for when everything hashes to the same bucket.